src/main/java/lotto/domain/Rank.java

```diff
@@ -0,0 +1,53 @@
+package lotto.domain;
+
+import java.util.Arrays;
+
+public enum Rank {
+
+    RANK_5(5000, 3),
+    RANK_4(50000, 4),
+    RANK_3(1500000, 5),
+    RANK_2(30000000, 5, true),
+    RANK_1(2000000000, 6),
+    RANK_OUT(0, 0);
+
+    private final long prize;
+    private final int winningLottoMatchCount;
+    private final boolean isMatchBonusNumber;
+
+    Rank(long prize, int winningLottoMatchCount) {
+        this.prize = prize;
+        this.winningLottoMatchCount = winningLottoMatchCount;
+        this.isMatchBonusNumber = false;
+    }
```

Comment:
this로 생성자를 호출할 수도 있어요!! 해당 부분은 참고해보시라고 작성해두겠습니다!! 👍🏼 

```suggestion
    Rank(long prize, int winningLottoMatchCount) {
        this(prize, winningLottoMatchCount, false);
    }
```